### PR TITLE
[refactor] Benutzung von FXML

### PR DESCRIPTION
**Pull request by joshuajeschek**

### CODE REVIEW

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 1]
```diff
@@ -1,71 +1,149 @@
...
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
 import javafx.scene.Scene;
 import javafx.scene.control.TextArea;
 import javafx.scene.image.Image;
+import javafx.scene.text.Text;
+import javafx.stage.FileChooser;
 import javafx.stage.Stage;
-import javafx.stage.WindowEvent;
+import javafx.stage.FileChooser.ExtensionFilter;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:18]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.stage.FileChooser.ExtensionFilter' import. Should be before 'javafx.stage.Stage'.

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 2]
```diff
@@ -1,71 +1,149 @@
...
 
 /**
  * Toolbox GUI.
  *
  */
 public class Main extends Application {
+    private Stage primaryStage;
+    static private ConfigClient configClient;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:26]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.modifier.ModifierOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'private' modifier out of order with the JLS suggestions.

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 3]
```diff
@@ -1,71 +1,149 @@
...
+    static private DataBaseClient dataBaseClient;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:27]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.modifier.ModifierOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'private' modifier out of order with the JLS suggestions.

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 4]
```diff
@@ -1,71 +1,149 @@
...
+     * PrintStream ps = new PrintStream(console, true);
+     * System.setOut(ps);
+     * System.setErr(ps);
+     * Scene terminal = new Scene(textArea);
+     * logStage.setScene(terminal);
+     * logStage.show();
+     * }
+     */
+
+     /**
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:74]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Block comment has incorrect indentation level 5, expected is 4, indentation should be the same level as line 78.

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 5]
```diff
@@ -1,71 +1,149 @@
...
+      * lädt Config aus JSON und versucht den Test zu starten.
+      * @param warnText
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:76]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.NonEmptyAtclauseDescriptionCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>At-clause should have a non-empty description.

Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:76]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.

File: src/main/java/github/weichware10/toolbox/AppController.java [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    @FXML
+    void startTrial() {
+        Logger.info("app:content Trying to start Trial");
+        main.startTrial(trialIdField.getText(), warnText);
+    }
+
+    @FXML
+    void resetDb(){
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/AppController.java:51]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '{' is not preceded with whitespace.

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 7]
```diff
@@ -1,15 +1,25 @@
...
+import github.weichware10.toolbox.gui.ConfirmBoxController;
+import github.weichware10.toolbox.gui.TestVorbildschirm;
+import github.weichware10.util.config.ConfigClient;
+import github.weichware10.util.db.DataBaseClient;
+import io.github.cdimascio.dotenv.Dotenv;
 import java.io.IOException;
 import java.io.OutputStream;
-import java.io.PrintStream;
+
+import com.fasterxml.jackson.databind.introspect.WithMember;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'com.fasterxml.jackson.databind.introspect.WithMember'

Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.fasterxml.jackson.databind.introspect.WithMember' import. Should be before 'java.io.OutputStream'.

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 8]
```diff
@@ -1,15 +1,25 @@
...
+
 import javafx.application.Application;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:13]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'javafx.application.Application'

File: src/main/java/github/weichware10/toolbox/gui/ConfirmBoxController.java [hunk 9]
```diff
@@ -0,0 +1,41 @@
+package github.weichware10.toolbox.gui;
+
+import github.weichware10.toolbox.Main;
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.FXML;
+import javafx.stage.Stage;
+
+public class ConfirmBoxController {
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/gui/ConfirmBoxController.java:9]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocTypeCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/github/weichware10/toolbox/gui/ConfirmBox.java [hunk 10]
```diff
@@ -1,69 +1,39 @@
 package github.weichware10.toolbox.gui;
 
-import javafx.event.ActionEvent;
-import javafx.event.EventHandler;
-import javafx.geometry.Insets;
-import javafx.geometry.Pos;
+import java.io.IOException;
+
+import javafx.fxml.FXMLLoader;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/gui/ConfirmBox.java:5]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'javafx.fxml.FXMLLoader'

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 11]
```diff
@@ -1,15 +1,35 @@
...
 import javafx.event.EventHandler;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
 import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
 import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+import javafx.scene.control.ButtonBar.ButtonData;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:28]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.scene.control.ButtonBar.ButtonData' import. Should be before 'javafx.scene.control.TextField'.

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 12]
```diff
@@ -50,20 +70,182 @@ public void handle(WindowEvent event) {
...
+                        "https://media.discordapp.net/attachments/707505654218358818/836645130417078282/TBY5IslL.png",
+                        "url2", "url3"));
+
+        createTestTrial(new Configuration(
+                "dunno yet",
+                "Test Question?",
+                zoomMapsConfiguration),
+                trialIdInput);
+    }
+    protected void createCodeChartsTestTrial(TextField trialIdInput) {
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:224]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.EmptyLineSeparatorCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'METHOD_DEF' should be separated from previous line.

File: src/main/java/github/weichware10/toolbox/AppController.java [hunk 13]
```diff
@@ -0,0 +1,89 @@
+package github.weichware10.toolbox;
+
+import java.io.IOException;
+
+import github.weichware10.util.Logger;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/AppController.java:5]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'github.weichware10.util.Logger'

Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/AppController.java:5]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.util.Logger' import. Should be before 'java.io.IOException'.

File: src/main/java/github/weichware10/toolbox/AppController.java [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
+import javafx.scene.control.TextField;
+import javafx.scene.control.ButtonBar.ButtonData;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/AppController.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.scene.control.ButtonBar.ButtonData' import. Should be before 'javafx.scene.control.TextField'.

File: src/main/java/github/weichware10/toolbox/gui/EndWindowController.java [hunk 15]
```diff
@@ -0,0 +1,75 @@
+package github.weichware10.toolbox.gui;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.ResourceBundle;
+
+import github.weichware10.toolbox.Main;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/gui/EndWindowController.java:7]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'github.weichware10.toolbox.Main'

Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/gui/EndWindowController.java:7]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.toolbox.Main' import. Should be before 'java.util.ResourceBundle'.

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 16]
```diff
@@ -1,86 +1,98 @@
...
 
         // Event welches beim schließen eines Fensters aufgerufen wird
         primaryStage.setOnCloseRequest(new EventHandler<WindowEvent>() {
             @Override
             public void handle(WindowEvent event) {
-                // Wir kümmern uns selber um das schließen
-                event.consume();
                 // Fenster schließen, ja oder nein?
-                closeProgramm(primaryStage);
+                boolean confirmation = new ConfirmDialog("Do you want to close the window?").getConfirmation();
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 111).

File: src/main/java/github/weichware10/toolbox/gui/dialogs/ConfirmDialog.java [hunk 17]
```diff
@@ -0,0 +1,65 @@
+package github.weichware10.toolbox.gui.dialogs;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import javafx.fxml.FXMLLoader;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/gui/dialogs/ConfirmDialog.java:6]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'javafx.fxml.FXMLLoader'

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 18]
```diff
@@ -1,86 +1,119 @@
...
+        PrintStream ps = new PrintStream(new Console(location, System.out), true);
+        System.setOut(ps);
+        System.setErr(ps);
+
         launch(args);
     }
 
+    /**
+     * setzt allgemeine Werte der primaryStage und startet die App.
+     * @param primaryStage - das Hauptfenster
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.

File: src/main/java/github/weichware10/toolbox/codecharts/CodeCharts.java [hunk 19]
```diff
@@ -0,0 +1,68 @@
...
+    @SuppressWarnings("unused")
+    private final ConfigClient configClient;
+    private final DataBaseClient dataBaseClient;
+    private final TrialData trialData;
+
+
+    /**
+     * Startet den CodeCharts-Versuch.
+     *
+     * @param primaryStage - hauptfenster
```
Comment (grenoui11e) [inline src/main/java/github/weichware10/toolbox/codecharts/CodeCharts.java:29]:
```suggestion
     * @param primaryStage - Hauptfenster
```

File: src/main/java/github/weichware10/toolbox/gui/dialogs/ConfirmDialog.java [hunk 20]
```diff
@@ -0,0 +1,83 @@
...
+     * @param prompt - Fragestellung
+     * @param icon - Zu verwendendes Icon, kann null sein
+     */
+    public ConfirmDialog(String prompt, String icon) {
+        this.prompt = prompt;
+        this.icon = icon;
+    }
+
+    /**
+     * zeigt den Dialog an und gibt einen Boolean-wert zurück.
```
Comment (grenoui11e) [inline src/main/java/github/weichware10/toolbox/gui/dialogs/ConfirmDialog.java:33]:
```suggestion
     * zeigt den Dialog an und gibt einen Boolean-Wert zurück.
```

File: src/main/java/github/weichware10/toolbox/gui/dialogs/ConfirmDialog.java [hunk 21]
```diff
@@ -0,0 +1,83 @@
...
+public class ConfirmDialog {
+
+    private final String prompt;
+    private final String icon;
+
+    /**
+     * Erstellt einen neuen ja/nein Dialog.
+     *
+     * @param prompt - Fragestellung
+     * @param icon - Zu verwendendes Icon, kann null sein
```
Comment (grenoui11e) [inline src/main/java/github/weichware10/toolbox/gui/dialogs/ConfirmDialog.java:25]:
```suggestion
     * @param icon - zu verwendendes Icon, kann null sein
```

File: src/main/java/github/weichware10/toolbox/zoommaps/ZoomMaps.java [hunk 22]
```diff
@@ -0,0 +1,67 @@
...
+    @SuppressWarnings("unused")
+    private final ConfigClient configClient;
+    private final DataBaseClient dataBaseClient;
+    private final TrialData trialData;
+
+
+    /**
+     * Startet den ZoomMaps-Versuch.
+     *
+     * @param primaryStage - hauptfenster
```
Comment (grenoui11e) [inline src/main/java/github/weichware10/toolbox/zoommaps/ZoomMaps.java:29]:
```suggestion
     * @param primaryStage - Hauptfenster
```

**Review by grenoui11e — APPROVED** [review]

File: src/main/java/github/weichware10/toolbox/eyetracking/EyeTracking.java [hunk 23]
```diff
@@ -0,0 +1,69 @@
+package github.weichware10.toolbox.eyetracking;
+
+import github.weichware10.toolbox.gui.End;
+import github.weichware10.util.ToolType;
+import github.weichware10.util.Logger;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/eyetracking/EyeTracking.java:5]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.util.Logger' import. Should be before 'github.weichware10.util.ToolType'.

File: src/main/java/github/weichware10/toolbox/codecharts/CodeCharts.java [hunk 24]
```diff
@@ -0,0 +1,68 @@
+package github.weichware10.toolbox.codecharts;
+
+import github.weichware10.toolbox.gui.End;
+import github.weichware10.util.ToolType;
+import github.weichware10.util.Logger;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/codecharts/CodeCharts.java:5]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.util.Logger' import. Should be before 'github.weichware10.util.ToolType'.

File: src/main/java/github/weichware10/toolbox/codecharts/CodeChartsController.java [hunk 25]
```diff
@@ -0,0 +1,36 @@
...
+import java.util.ResourceBundle;
+import javafx.fxml.FXML;
+
+/**
+ * Controller for {@link CodeCharts}.
+ */
+public class CodeChartsController {
+
+    @FXML
+    private ResourceBundle resources;
```
Comment (Rotwolf) [inline src/main/java/github/weichware10/toolbox/codecharts/CodeChartsController.java:14]: wozu resurces und location?

Comment (joshuajeschek) [reply src/main/java/github/weichware10/toolbox/codecharts/CodeChartsController.java:14]: wird defaultmäßig von scene builder generiert, ich würde es erstmal überall drin stehen lassen

File: src/main/java/github/weichware10/toolbox/gui/App.java [hunk 26]
```diff
@@ -0,0 +1,281 @@
...
+        if (dataBaseClient == null) {
+            controller.setWarnText("no database connection");
+            return;
+        }
+
+        ZoomMapsConfiguration zoomMapsConfiguration = new ZoomMapsConfiguration(
+                4.2f,
+                true,
+                Arrays.asList(
+                        "https://media.discordapp.net/attachments/707505654218358818/836645130417078282/TBY5IslL.png",
```
Comment (Rotwolf) [inline src/main/java/github/weichware10/toolbox/gui/App.java:154]: was macht diese Funktion, wird sie momentan überhaupt aufgerufen (ist sie vom "Nutzer" im Moment Aufrufbar) und warum greift sie auf Discord zu xD

Comment (joshuajeschek) [reply src/main/java/github/weichware10/toolbox/gui/App.java:154]: das ist die funktion, wo von dem Admin-Menü aus ein neuer Test erstellt werden kann, der link ist ein link zu einem Bild von einem süßen Häschen 🥺
ist nur von admins aufrufbar damit man einen test ausprobieren kann

**Review by Rotwolf — APPROVED** [review]
Kleine Verständnissfragen, sonst Topf! ohne f

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 27]
```diff
@@ -1,86 +1,83 @@
...
-import github.weichware10.toolbox.gui.ConfirmBox;
-import github.weichware10.toolbox.gui.Startbildschirm;
-import java.io.IOException;
-import java.io.OutputStream;
+import github.weichware10.toolbox.gui.App;
+import github.weichware10.toolbox.gui.dialogs.ConfirmDialog;
+import io.github.cdimascio.dotenv.Dotenv;
 import java.io.PrintStream;
+
+import org.joda.time.DateTime;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:8]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'org.joda.time.DateTime'

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 28]
```diff
@@ -1,86 +1,83 @@
...
+
 import javafx.application.Application;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:10]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.application.Application' import. Should be before 'org.joda.time.DateTime'.

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 29]
```diff
@@ -1,86 +1,83 @@
...
-import javafx.event.EventHandler;
-import javafx.scene.Scene;
-import javafx.scene.control.TextArea;
+import javafx.geometry.Rectangle2D;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.geometry.Rectangle2D' import. Should be before 'org.joda.time.DateTime'.

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 30]
```diff
@@ -1,86 +1,83 @@
...
 import javafx.scene.image.Image;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.scene.image.Image' import. Should be before 'org.joda.time.DateTime'.

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 31]
```diff
@@ -1,86 +1,83 @@
...
+import javafx.stage.Screen;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:13]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.stage.Screen' import. Should be before 'org.joda.time.DateTime'.

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 32]
```diff
@@ -1,86 +1,83 @@
...
 import javafx.stage.Stage;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.stage.Stage' import. Should be before 'org.joda.time.DateTime'.

File: src/main/java/github/weichware10/toolbox/Main.java [hunk 33]
```diff
@@ -1,86 +1,83 @@
...
 import javafx.stage.WindowEvent;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/Main.java:15]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.stage.WindowEvent' import. Should be before 'org.joda.time.DateTime'.

File: src/main/java/github/weichware10/toolbox/gui/dialogs/LogDialog.java [hunk 34]
```diff
@@ -0,0 +1,33 @@
...
+
+import github.weichware10.toolbox.Console;
+import github.weichware10.util.Logger;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
+
+public class LogDialog {
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/gui/dialogs/LogDialog.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocTypeCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/github/weichware10/toolbox/gui/dialogs/LogDialog.java [hunk 35]
```diff
@@ -0,0 +1,33 @@
...
+    public LogDialog() {
```
Comment (weichbot10) [inline src/main/java/github/weichware10/toolbox/gui/dialogs/LogDialog.java:12]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocMethodCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

**Pull request merged**